src/patternfly/components/Page/page.scss

```diff
@@ -92,13 +95,27 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
   --pf-c-page__main-section--xl--PaddingRight: var(--pf-global--spacer--lg);
   --pf-c-page__main-section--xl--PaddingBottom: var(--pf-global--spacer--lg);
   --pf-c-page__main-section--xl--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-page__main-section--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
-  --pf-c-page__main--ZIndex: var(--pf-global--ZIndex--xs);
   --pf-c-page__main-breadcrumb--main-section--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-page__main-section--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-page__main-section--m-light--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-page__main-section--m-dark-100--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-100);
+  --pf-c-page__main-section--m-dark-200--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
 
   // Limit width
   --pf-c-page--section--m-limit-width--MaxWidth: calc(#{pf-size-prem(2000px)} - var(--pf-c-page__sidebar--Width));
 
+  // Sticky
+  --pf-c-page--section--m-sticky--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-page--section--m-sticky--BoxShadow: var(--pf-global--BoxShadow--sm-bottom);
+  --pf-c-page--section--m-sticky-bottom--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-page--section--m-sticky-bottom--BoxShadow: var(--pf-global--BoxShadow--sm-top);
+
+  // Shadows
+  --pf-c-page--section--m-shadow--BoxShadow: var(--pf-global--BoxShadow--sm-bottom);
```

Comment:
same comment as above i think itd be easier to follow for the dev if `m-shadow` is `m-shadow-bottom` so that it's opposite of `m-shadow-top`